server/ansible/roles/pbench-server-activate-start-httpd/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# we need the top-level pbench directory on each server
+# in order to create a bunch of links in the httpd doc root
+# (see the last task in this file). We need to ask the server
+# for it because it may vary by server.
+
+- name: fetch the pbench directory from remote
+  shell: getconf.py pbench-top-dir pbench-server
```

Comment:
Why isn't this variable fetch part of the `pbench-server-vars` role?

Reply:
Historical reasons. I'll consolidate and simplify.